docker/set_up_awscli.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Install aws cli as the current user in /tmp so it doesn't interfere with
+# socorro/system requirements
+HOME=/tmp pip install awscli --user
+
+# Fix permissions so anyone can use it
+find /tmp/.local/ -type d -exec '/bin/chmod' '755' '{}' ';'
+find /tmp/.local/ -type f -exec '/bin/chmod' '644' '{}' ';'
+chmod 755 /tmp/.local/bin/*
```

Comment:
This has a lot of "ugh" in it. The aws cli uses a different version of boto and friends than what we use in Socorro. So I figured I'd "install" it in such a way that it doesn't affect everything else.

The best way I could figure to do that was to install it for a "user", but the app user doesn't have a home directory, so I set `HOME` to `/tmp`. Then it installs stuff in weird places and with permissions that don't let anyone else on the box use it. Thus the permissions fixes.

I'm game for a better way to fix permissions in a directory tree. Particularly the file ones--that's goofypants.